### ISSUE TITLE
v5.1.5 lightning import table schema with clustered index  to v4.0.16 fail 

### ISSUE THREAD

**Issue by seiya-annie** [pingcap/tidb#40058]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

1. install upstream with v4.0.16 version
2. install downstream with v5.1.5 version
3. use v5.1.5 lightning import table to upstream:
```sql
CREATE TABLE `sbtest1` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `k` int(11) NOT NULL DEFAULT '0',
  `c` char(120) NOT NULL DEFAULT '',
  `pad` char(60) NOT NULL DEFAULT '',
  PRIMARY KEY (`id`) /*T![clustered_index] CLUSTERED */,
  KEY `k_1` (`k`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin AUTO_INCREMENT=391701;
```
### 2. What did you expect to see? (Required)
import successfully
### 3. What did you see instead (Required)
```code
2022-12-17T09:13:42.546Z	INFO	host/host.go:30	Execute command	{"command": "/tidb-lightning \"-backend\" \"local\" \"-sorted-kv-dir\" \"/tmp/sorted-kv-dir\" \"-d\" \"s3://nfs/jianxu/test-data?access-key=minioadmin&secret-access-key=minioadmin&endpoint=http%3a%2f%2fminio.pingcap.net%3a9000&force-path-style=true\" \"-pd-urls\" \"upstream-pd.brie-acceptance-compatiblity-tps-1327240-1-662:2379\" \"-tidb-host\" \"upstream-tidb.brie-acceptance-compatiblity-tps-1327240-1-662\" \"-tidb-port\" \"4000\" \"-tidb-user\" \"root\" \"-tidb-password\" \"\" \"-c\" \"/tmp/tidb-lightning.toml\"", "timeout": "30m", "resource name": "lightning"}
Verbose debug logs will be written to /tmp/lightning.log.2022-12-17T09.13.42Z

Error: restore table schema `clustered`.`t1` failed: run create schema job failed: Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 140 near "CLUSTERED,INDEX `idx`(`a`)) ENGINE = InnoDB DEFAULT CHARACTER SET = UTF8MB4 DEFAULT COLLATE = UTF8MB4_BIN AUTO_RANDOM_BASE = 3143025;" 
tidb lightning encountered error:  restore table schema `clustered`.`t1` failed: run create schema job failed: Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your TiDB version for the right syntax to use line 1 column 140 near "CLUSTERED,INDEX `idx`(`a`)) ENGINE = InnoDB DEFAULT CHARACTER SET = UTF8MB4 DEFAULT COLLATE = UTF8MB4_BIN AUTO_RANDOM_BASE = 3143025;" 
```
### 4. What is your TiDB version? (Required)

/ # ./tidb-lightning -V
Release Version: v5.1.5
Git Commit Hash: 1a128171cb65d68e109073bbabb4e1c1cf9e5e3c
Git Branch: heads/refs/tags/v5.1.5
Go Version: go1.16.4
UTC Build Time: 2022-12-16 11:05:22
Race Enabled: false
/ #

**Comment by seiya-annie** [comment]
[sbtest.sbtest1-schema.sql.txt](https://github.com/pingcap/tidb/files/10266538/sbtest.sbtest1-schema.sql.txt)

**Comment by gozssky** [comment]
Duplicate with https://github.com/pingcap/br/issues/1362.

**Comment by seiya-annie** [comment]
this issue will not be fixed on v5.1.5. 
customer can use latest lighting to import data.
